### DISCUSSION TITLE
Point Cloud data Importation in Model Library

### DISCUSSION THREAD

**Discussion by agrignard** [gama-platform/gama#1097] [Ideas]
**Describe the improvement you'd like**
In the data importation library it would be nice to have an example to load point cloud data (e.g https://pointclouds.org/documentation/tutorials/pcd_file_format.html) 

A kind of point cloud data is used in this model ```Toy Models/Art/Radiohead In Point Cloud/models/Radiohead In Point Coud.gaml```but it's hand made and doesn't use any point_cloud_file structure it just instantiating agent in a 3D space according to x,y,z,color read in a csv file. 

**Describe alternatives you've considered**
An alternative with a ```point_cloud_file```struvture in the same way we have ```dem_file```

**Additional context**
Then it can be somehow link to the ```mesh``` concept to visualize it

**Comment by AlexisDrogoul** [comment]
@agrignard any progress on this one ? If not, I will move it to the discussions section.
